cmd/run.go

```diff
@@ -16,6 +16,8 @@ func newRun(uii *ui.UI, contextProvider contextProvider) *cobra.Command {
 		validateArgs bool
 	)
 
+	executorRegistry := executors.NewExecutorRegistry(executors.ShellExecutor)
```

Comment:
If the factory function is renamed then the variable should be called `registry`